api/backend/articles/routes.py

```diff
@@ -0,0 +1,69 @@
+from .. import api, db
+from ..articles.decorators import article_exists, valid_article_form
+from ..articles.utils import create_article, update_article
+from ..articles.schemas import article_schema, articles_schema 
+from ..models import Article
+from flask import Blueprint, request
+from flask_restful import Resource
+
+# Blueprint for articles
+articles_bp = Blueprint("articles", __name__)
+
+
+# Class to handle information regarding the Article model
+class ArticleCreate(Resource):
```

Comment:
Rename this to Articles since we are not just creating an Article

Reply:
Do you mean class ArticlesCreate(Resource)?

Reply:
Yes. Rename `class ArticleCreate` to class Articles